src/leiningen/new/xiana.clj

```diff
@@ -1,46 +1,57 @@
 (ns leiningen.new.xiana
-  (:require [leiningen.new.templates :refer [multi-segment sanitize-ns renderer name-to-path ->files]]
-            [leiningen.core.main :as main]))
+  (:require
+    [clojure.set :as set]
+    [clojure.string :as st]
```

Comment:
A little point: Something we don't want.

Reply:
I've change it on 'clset' and 'clstring'